runtime/basilisk/src/xcm.rs

```diff
@@ -256,4 +256,5 @@ pub type LocalAssetTransactor = MultiCurrencyAdapter<
 	LocationToAccountId,
 	AssetId,
 	CurrencyIdConvert,
+	(),
```

Comment:
Let's create an issue for this one to not forget - I will need to look into this new param.

Reply:
yes, more info -> https://github.com/open-web3-stack/open-runtime-module-library/pull/675